### ISSUE TITLE
a11y issue:  unable to close modal window with keystrokes. 

### ISSUE THREAD

**Issue by cymruhangu** [cymruhangu/thinkful-tube#7]
As part of this issue.  The X positioning needs to be fixed so that it is always visible no matter the screensize.

**Comment by cymruhangu** [comment]
Added event listener on ESC in the modal-container.  To close the modal window you can tab or shift-tab out of the native YouTube controls to leave the iframe and then press ESC to close the window.  To me this should satisfy the accessibility objectives of this project but will discuss with Marc on Tuesday before losing issue.

**Comment by cymruhangu** [comment]
Voiceover now reads native Youtube buttons and links and gives accurate instructions for entering and exiting window.  An aria label was added which allows Voiceover to speak instructions for exiting the player.
